docs/FAQ/FAQ-Wallet.md

```diff
@@ -1,133 +1,80 @@
-* Zap - Lightning Network LND wallet
-* Spark - Lightning Network c-lightning wallet
-* Sentinel - Watch-only on-chain wallet for Android
-* ArcBit / bitWallet - Watch-only on-chain wallet for Android
-
-## How to connect my wallet to BTCPay Server?
-
-The process of connecting a wallet to your BTCPay server is the same for both third-party or self-hosted deployments. Your private keys are never uploaded nor required by the BTCPay Server. You’re only uploading the xpubkey.
-
-There are two ways to connect your BTCPay and your wallet:
-
-1. Automatic with Ledger Nano S hardware wallet
-2. Manual with any wallet supporting xpubkey (Electrum highly recommended)
-
-Watch the video below or read [this article](https://bitcoinshirt.co/how-to-create-store-accept-bitcoin/8/#Connecting-BTCPay-with-your-wallet) for step by step instructions.
-
-[![ConnectBTCPayWallet](https://img.youtube.com/vi/xX6LyQej0NQ/mqdefault.jpg)](https://www.youtube.com/watch?v=xX6LyQej0NQ "BTCPay Server - Connecting Wallet")
+However, it's recommended to use a built in wallet for funds management. The built in wallet improves your privacy by default, but also solves user-experience issues like [gap-limit](#missing-payments-in-my-software-or-hardware-wallet).
 
 ## Missing payments in my software or hardware wallet
 
-If your internal BTCPay wallet is showing the transactions and you do not see the funds on your desktop, mobile or hardware wallet, you need to **increase the gap limit** of the external wallet you're using. Do not worry, your funds are safe, they're just not showing yet.
-
-Most wallets, have a gap limit of 20. This means that after 20 consecutive unpaid invoices, wallet will stop fetching the transactions beyond that and won't show them. The solution is to increase the gap limit. Not all wallets have this feature.
-
-For that reason, it's highly recommended that you use [Electrum wallet](https://electrum.org/). To set increase the gap limit in Electrum, [follow this video](https://www.youtube.com/watch?v=Fi3pYpzGmmo)
+If you're using an [existing software or a hardware wallet](/WalletSetup/#use-an-existing-wallet) with your BTCPay Server, you may experience a discrepancy between balance in your BTCPay wallet and the external wallet's web, destkop or mobile app. This discrepancy is usually related to a **gap-limit** issue. 
 
-Enter following commands in Electrum console:
+### The gap limit problem
 
-```
- wallet.change_gap_limit(100)
- wallet.storage.write()
-```
-Restart your Electrum and verify that the newly set gap limit is correct by entering:
+Majority of third party wallets are [SPV wallets](https://en.bitcoinwiki.org/wiki/Simplified_Payment_Verification), which share a node between many users. To prevent performance issues, SPV wallets limit the amount of addresses without balance they follow on the blockchain. BTCPay Server generates a new address for every invoice. 
```

Comment:
```suggestion
The majority of third party wallets are [SPV wallets](https://en.bitcoinwiki.org/wiki/Simplified_Payment_Verification), which share a node between many users. To prevent performance issues, SPV wallets limit the amount (typically 20) of addresses without balance that they track on the blockchain. BTCPay Server generates a new address for every invoice. 
```